### DISCUSSION TITLE
Wrong: "Your GitHub Copilot trial ends in 3 days"

### DISCUSSION THREAD

**Discussion by gitcnd** [community/community#30165] [Copilot Conversations]
I followed the instructions in the 18th Aug email: Discover the “most mind-blowing application of ML” 
...which purported to give me another month of service.

Today (2 days later), I got another email still saying "Your access to GitHub Copilot will expire on August 22, 2022."

Someone needs to fix something, because dates are not getting updated when we followed the 18th Aug set of instructions.

**Comment by Zoltus** [comment]
That is a bug confirmed by github employee and shouldnt worry about it.

![image](https://user-images.githubusercontent.com/23402980/185735202-a4a8bbc0-f830-45c7-a3aa-292a30eb0c1e.png)

**Comment by shanen5** [comment]
My experiences are somewhat different, but I'd like to start with the fundamental rudeness of sending that email from a no-reply email address. Score points for rudeness.

That triggered my next response of unsubscribing. Not enough mutual interest for you to ask why, though I am obviously willing to say more (per this comment). Minor rudeness, but I'll note that my experience with sales was that I should at least act like I cared about the customers' needs and problems and even feelings.

Which leads to the topic of sales and how to avoid them. The pitch to buy Copilot falls really flat if anyone or even anything (as in a computer) had been paying attention. My free trial did not go well. Now when I arrived over here at GitHub at least the system offered the annotation that I never finished the installation of Copilot. Maybe someone should have noticed that before asking me for money? Even better, ask me what part of the setup stopped me. Too bad considering how much I had been looking forward to trying the tool during that long waiting period.

Not that I think anyone at GitHub cares (which must reveal something about a broken financial model), but I'll throw in what little I can recall. It must have seemed overwhelmingly complicated so it scared me off somewhere quite early in the process. If not, then I should have a concrete memory of some struggles that didn't go well.

Have a nice day, but no money for you.

**Comment by gitcnd** [comment]
"Here here" regarding the "no-reply" email - the service I run charges way less than 1/3 of what github do for this, and even *I* employ human support for all customers at the "reply" end of everything I send out!

It's also made me irritated that they've taken the *licensed* intellectual-property open-source contributions of myself and everyone else that they trained their robot on, generated a derivative product from everything we were kind enough to give away for free (and mostly under the restriction that derivative works also be free and open!), and contrary to the wishes and licensing that we all spent pains to select*, they're now providing our own work back to us in this derivative product they've built FOR MONEY.  They even have the cheek to ask us to allow their robot to *continue* training itself on what we do with their product after we pay them too!!

And to rub salt into the wound, they're giving this away for free to "popular" open-source contributors as if somehow the popularity of what someone decides to contribute to the world is a fair measure of whether or not they should be billed for this.

Here's a suggestion - how about EVERY SINGLE PERSON who's source code contributed to this product, be granted a perpetual free license to use it?!

* I would go as far as to state that there is a legal duty for this to be the case, since the majority of licensing that applies to the projects used in the training of this tool carry terms preventing the sale of works based on them, along with other terms including attribution, source code release, licensing and so on, and for every project that does not carry such a license, the fact that it was GitHub themselves who gave us the means to select a license that they (secretly at the time) knew would benefit their own later exploitation of our efforts is sufficiently bad-faith that such non-explicitly-stated terms should in fact apply nonetheless.

**Comment by kestasjk** [comment]
I think copilot is a great feature; glad to have access to a tool that makes coding easier for me, and if they used my code to help create it I got to use a world class, ad-free social software development platform for free for years.. Can't really complain.

What I can say though is it says "You have 3 days left on your GitHub Copilot trial." I thought I subscribed to the personal plan but I guess not, but I can't see how I can subscribe. If I google for Copilot and go to the page it just talks about the free 60 day trial, not sure where I go to pay the $10/mo.
